markdown/dev/reference/api/path/settext/en.md

```diff
@@ -7,7 +7,7 @@ The `Path.setText()` method sets text on the path.
 ## Signature
 
 ```js
-Path path.setText(string text, string className)
+Path path.setText(string text[, string className])
```

Comment:
Same remark. No need for the square brackets as the optional value has a default assigned.